packages/react-native/Libraries/Components/ScrollView/ScrollView.d.ts

```diff
@@ -405,7 +405,7 @@ export interface ScrollViewPropsIOS {
 
   /**
    * This property specifies how the safe area insets are used to modify the content area of the scroll view.
-   * The default value of this property must be 'automatic'. But the default value is 'never' until RN@0.51.
+   * The default value of this property is "never". Available on iOS 11 and later.
```

Comment:
Current versions of RN don't support anything before 13.4, or maybe even 15, so this couldn't just be removed.

Reply:
Thanks for the feedback @NickGerleman! I've updated both the JS and TS documentation to remove the iOS specific version part